### ISSUE TITLE
VTK output for fields on continuous functionspaces is broken

### ISSUE THREAD

**Issue by stephankramer** [thetisproject/thetis#208]
It seems that for any field that is defined on a continuous function space the corresponding vtk output produces broken vtu files. I think this is due to the self-proclaimed hack (line 123 of thetis/exporter.py) to avoid storing a separate dg coordinates function in every `File` object. If I comment out that line, things seem to work again. This is presumably because vtu files with only continuous fields do not have a discontinuous mesh. Is the fix to just add a `and not is_cg(function)` ?

Reproducer:
```
from thetis import *
from thetis.exporter import ExportManager
  
mesh = UnitSquareMesh(2,2)

function_spaces = [
        FunctionSpace(mesh, "DG", 0, name="P0"),
        FunctionSpace(mesh, "CG", 1, name="P1"),
        FunctionSpace(mesh, "CG", 2, name="P2"),
        FunctionSpace(mesh, "DG", 1, name="P1DG"),
        FunctionSpace(mesh, "DG", 2, name="P2DG")]

fields = {}
field_meta_data = {}
for fs in function_spaces:
    name = fs.name + "_field"
    field = Function(fs, name=name)
    fields[name] = field
    field_meta_data[name] = {'shortname': name, 'filename': name}

em = ExportManager('outputs', fields.keys(), fields, field_meta_data, export_type='vtk')
em.export()
```

**Comment by tkarna** [comment]
Yes, the bug is indeed caused by the coordinate caching in `ExportManager` which currently is only implemented for DG function space. We aspire to do the caching in Firedrake instead (see firedrakeproject/firedrake/pull/1744).

**Comment by tkarna** [comment]
Fixed in #215
